buildroot-external/busybox.config

```diff
@@ -951,7 +951,7 @@ CONFIG_PING6=y
 # CONFIG_ROUTE is not set
 # CONFIG_SLATTACH is not set
 # CONFIG_SSL_CLIENT is not set
-CONFIG_TC=y
+# CONFIG_TC is not set
```

Comment:
You were not correct [here](https://github.com/home-assistant/operating-system/pull/3767#discussion_r1899152223), `CONFIG_TC` is part of the Busybox config and enables the `tc` (traffic control) command. I'm not exactly sure where/if it's used but definitely this needs to be looked into.

Reply:
Looks like busybox needs a patch
https://github.com/mirror/busybox/pull/96
https://lists.openembedded.org/g/openembedded-core/message/201449

Reply:
https://github.com/home-assistant/buildroot/pull/68